index.js

```diff
@@ -94,25 +186,39 @@
 //70s should be Cs 
 //60s should be D 
 //and anything below 60 should be F
-  
-
-  
-  
+function grade(percent) {
+    if (percent >= 90) {
+        return 'A';
+    } else if (percent >= 80) {
+        return 'B';
+    } else if (percent >= 70) {
+        return 'C';
+    } else if (percent >= 60) {
+        return 'D';
+    } else {
+        return 'F';
+    }
+}
+console.log(grade(94));
 
 /************************************************************** Stretch **************************************************************/
 //Create a function that counts the number of vowels within a string. It should handle both capitalized and uncapitalized vowels.
 // Hint - you may need to study tomorrow's traning kit on arrays 
 // try looking up the .includes() method
-
-
-
-
+function countVowels(input) {
+    // I read up on the .includes() method, and think that it's a nice way to do it. But, I initially wrote it like this, so hopefully that is okay. 
+    let array = Array.from(input.toLowerCase());
+    let vowelCount = 0;
+
+    for (i = 0; i < array.length; i++) {
+        if (array[i] === 'a' || array[i] === 'e' || array[i] === 'i' || array[i] === 'o' || array[i] === 'u') {
+            vowelCount++;
+        }
+    }
+    return vowelCount;
+}
+console.log(countVowels("Hello world. How are yOu?"));
```

Comment:
The main benefit of using `.includes()` would be being able to shorten your code, but this method is completely fine as well. It looks great 🙂